ci-operator/step-registry/mco/conf/day2/destroy-custom-mcp/mco-conf-day2-destroy-custom-mcp-commands.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function set_proxy () {
+    if [ -s "${SHARED_DIR}/proxy-conf.sh" ]; then
+        echo "Setting the proxy ${SHARED_DIR}/proxy-conf.sh"
+        # shellcheck source=/dev/null
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "No proxy settings"
+    fi
+}
+
+function run_command() {
+    local CMD="$1"
+    echo "Running command: ${CMD}"
+    eval "${CMD}"
+}
+
+# waiting for mcp become a given status
+function wait_for_config_to_be_applied() {
+    local mcp_name="$1"
+    local expected_machine_count="$2"
+    local out MACHINECOUNT 
+
+    echo "Waiting for $mcp_name MachineConfigPool to finish updating..."
```

Comment:
Suggest to add one more step to see the mcp start updating:
```
  echo "Waiting for $mcp_name MachineConfigPool to start updating..."
  oc wait mcp "$mcp_name" --for='condition=UPDATING=True' --timeout=300s &>/dev/null
```

Reply:
Ignore it, I saw it on https://github.com/openshift/release/pull/55328/files#diff-6da6be9bbf08fa8f36b9c7df36e496571acc8899a4e116eff4494a8a255b9c85R69

Reply:
do you mean extract these 2 lines to a new method?

Reply:
ok